info.colobot.Colobot.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">info.colobot.Colobot.desktop</id>
```

Comment:
The appdata lacks some required information like: OARS, releases

Reply:
It should be pushed upstream too.

Reply:
I'll be contacting upstream some time tomorrow about adding appdata and more